src/pykiso/test_coordinator/assert_step_report/assert_step_report.py

```diff
@@ -96,13 +97,30 @@ def _get_variable_name(f_back: types.FrameType, assert_name: str) -> str:
     # Get current line number
     line_no = f_back.f_lineno
 
-    # Get line content, if statement not
-    # complete, read from bottom to up
+    # The inspect module returns the first line in python >=3.8 and the last line in python 3.7
+    first_line = sys.version_info.major > 3 or sys.version_info.minor >= 8
```

Comment:
```suggestion
    first_line = sys.version_info > (3, 8)
```
One of multiple reasons why python's so cool :D

Reply:
Thats nice!